interoperability-tests/Testing-between-Gramine-and-IntelSGXSDK.md

```diff
@@ -0,0 +1,147 @@
+Testing between Gramine and Intel SGX SDK RA-TLS
+====
+
+# Introduction
+
+This document records the interoperability testing between [Gramine](https://github.com/gramineproject/gramine) and [Intel SGX SDK](https://github.com/intel/linux-sgx), both of which have implemented Interoperable RA-TLS.
+
+We conducted two sets of tests, in which Gramine and Intel SGX SDK RA-TLS respectively verified each other's certificates and established TLS secure channels. You can verify our tests by following the steps below.
+
+# Prerequisites
+
+## Intel SGX SDK and PSW packages
+
+To run the tests, first you need to download and install Intel SGX SDK and corresponding PSW packages on your system. 
+
+Interoperable RA-TLS was implemented since [Intel SGX SDK 2.22](https://github.com/intel/linux-sgx/releases/tag/sgx_2.22) 
```

Comment:
Please add a dot (`.`) at the end of this sentence.

Reply:
updated